modules/Staff/applicationForm_manage_accept.php

```diff
@@ -121,250 +129,260 @@
                 echo __('Step')." $step";
                 echo '</h3>';
 
-                if ($values['gibbonPersonID'] == '') { //USER IS NEW TO THE SYSTEM
-                    $informApplicant = 'N';
-                    if (isset($_POST['informApplicant'])) {
-                        if ($_POST['informApplicant'] == 'on') {
-                            $informApplicant = 'Y';
-                            $informApplicantArray = array();
-                        }
-                    }
 
-                    //DETERMINE ROLE
-                    $gibbonRoleID = ($values['type'] == 'Teaching') ? '002' : '006';
+                $createStaffAccount = false;
+                if (isset($_POST['createStaffAccount'])) {
+                    if ($_POST['createStaffAccount'] == 'on') {
+                        $createStaffAccount = true;
+                    }
+                }
```

Comment:
This could be simplified into a single line, since checkboxes are empty when not set.

```suggestion
                $createStaffAccount = !empty($_POST['createStaffAccount']);
```